python/ray/util/actor_group.py

```diff
@@ -49,6 +50,7 @@ def remote(self, *args, **kwargs):
         ]
 
 
+@Deprecated(message="Use ray.util.multiprocessing instead.")
```

Comment:
Should we also mention BatchPredictor per discussion?

Reply:
yeah, per @pcmoritz's comment

> I expect the deprecation of ActorPool and ActorGroup to be done much better -- there should be a clear guideline on what users should use instead, together with examples in the documentation on how to translate existing code to the new model. In particular, the question of whether we recommend multiprocessing or Datasets / BatchPredictor as a replacement needs answering.

Reply:
Added, PTAL